src/System.Collections.Immutable/tests/ImmutableArrayTest.cs

```diff
@@ -1134,26 +1128,7 @@ public void IndexGetter()
             Assert.Throws<InvalidOperationException>(() => ((IList<int>)s_emptyDefault)[0]);
             Assert.Throws<InvalidOperationException>(() => ((IReadOnlyList<int>)s_emptyDefault)[0]);
         }
-
-        [Fact]
-        public void ExplicitMethods()
-        {
-            IList<int> c = s_oneElement;
-            Assert.Throws<NotSupportedException>(() => c.Add(3));
-            Assert.Throws<NotSupportedException>(() => c.Clear());
-            Assert.Throws<NotSupportedException>(() => c.Remove(3));
-            Assert.True(c.IsReadOnly);
-            Assert.Throws<NotSupportedException>(() => c.Insert(0, 2));
-            Assert.Throws<NotSupportedException>(() => c.RemoveAt(0));
-            Assert.Equal(s_oneElement[0], c[0]);
-            Assert.Throws<NotSupportedException>(() => c[0] = 8);
-
-            var enumerator = c.GetEnumerator();
-            Assert.True(enumerator.MoveNext());
-            Assert.Equal(s_oneElement[0], enumerator.Current);
-            Assert.False(enumerator.MoveNext());
-        }
```

Comment:
You removed this because every one of these asserts is already covered elsewhere?